src/frontend/Komponenter/Personoversikt/Infotrygdperioderoversikt.tsx

```diff
@@ -53,10 +57,13 @@ const InfotrygdEllerSummertePerioder: React.FC<{ perioder: InfotrygdPerioderResp
 
     return (
         <>
-            <StyledAlertStripe>
-                Denne siden viser vedtakshistorikk fra EV VP. (Saker før desember 2008 - PE PP må
-                sjekkes manuelt i Infotrygd)
-            </StyledAlertStripe>
+            <FlexBox>
+                <StyledAlertStripe>
+                    Denne siden viser vedtakshistorikk fra EV VP. (Saker før desember 2008 - PE PP
+                    må sjekkes manuelt i Infotrygd)
+                </StyledAlertStripe>
+                <Historiskpensjon fagsakPersonId={fagsakId} />
```

Comment:
vi kaller variabelen for fagsakPersonId, men sender inn fagsakId - én av de må renames tror jeg?